span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanNormalizer.java

```diff
@@ -277,4 +285,28 @@ private Event buildEvent(
 
     return eventBuilder.build();
   }
+
+  private void logSpanConversion(Span jaegerSpan, RawSpan rawSpan) {
+    if (LOG.isDebugEnabled()) {
+      try {
+        LOG.debug("Converted Jaeger span: {} to rawSpan: {} ", jaegerSpan, convertToJsonString(rawSpan, rawSpan.getSchema()));
+      } catch (IOException e) {
+        LOG.warn("An exception occurred while converting avro to JSON string", e);
+      }
+    }
+  }
+
+  // We should have a small library for useful short methods like this one such as this.
+  public static <T extends SpecificRecordBase> String convertToJsonString(T object, Schema schema)
```

Comment:
Can you please attach a sample output here so that we can see how that looks?

Reply:
I mean, it's a JSON string representing the rawspan. The output is too verbose.